src/utils/test.ts

```diff
@@ -0,0 +1,31 @@
+import { isTestingEnv } from '~/environment';
+import { E2E_PREFIX } from '~/testing/constants';
+/**
+ * This function take either a string as parameter or an object and returns
+ * test attributes if env.NODE_ENV is dev, or env.E2E_TEST_MODE is set to true.
+ *
+ * If it gets a string, it returns an object with the common data attributes
+ * we use to store id. If it takes an object, it returns the same object with
+ * all the keys prefixed with the suffix we use for e2e tests.
+ *
+ * @example
+ * // returns { "data-test": "my-value" }
+ * getTestAttributes("my-value");
+ * @example
+ * // returns { "data-test-my-attribute": "my-value" }
+ * getTestAttributes({ "my-attribute": "my-value "});
+ */
+export const getTestAttributes = (
```

Comment:
This function by its purpose cannot be a part of `utils`

Reply:
You prefer `src/testing/utils`? Or `src/testing/helpers`?